projects/storefrontapp-e2e-cypress/cypress/helpers/applied-promotions.ts

```diff
@@ -67,9 +78,9 @@ export function selectPaymentMethod() {
     .should('not.be.empty');
   cy.get('.cx-card-title').should('contain', '✓ DEFAULT');
   cy.get('.card-header').should('contain', 'Selected');
+  waitForPage('/checkout/review-order', 'reviewOrder');
```

Comment:
I'd like this to store the alias in a variable to be used in line 83

For example: https://github.com/SAP/spartacus/pull/12440/files#diff-091683f038132765619a1bc2f98b544dc6c2ecc944246c89595ef1db3c5010afR19

Reply:
Done